apps/yapms/src/lib/components/modals/stylemodal/StyleModal.svelte

```diff
@@ -10,6 +10,13 @@
 			open: false
 		});
 	}
+
+	function toggleInsets() {
+		const svg = document.getElementById('map-div')?.querySelector<SVGElement>('svg');
+		if (svg) {
+			svg.classList.toggle('insetsHidden');
+		}
+	}
```

Comment:
The modals themselves shouldn't directly modify the DOM, will explain in the final review.